stl/inc/optional

```diff
@@ -53,12 +53,15 @@ protected:
     _THROW(bad_optional_access{});
 }
 
+#if _HAS_CXX17

```

Comment:
We should scratch this, as it is a remnant of the inclusion in a non C++17 header